config/config.go

```diff
@@ -72,6 +76,28 @@ type Config struct {
 	Plugin map[string]Plugin `json:"plugin,omitempty"`
 }
 
+func (c *Config) Validate(r io.Reader) error {
```

Comment:
Wouldn't `Decode` or `Unmarshall` be more fitting?

Reply:
Good point. I'm even considering `DecodeYAML` to signal that we want a reader that we can read yaml from?!

Reply:
Yeah, I think DecodeYAML would be even more fitting